<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cctvlabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CCTV MASKING</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1011</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Image Capture</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>441</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Load</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>FileRoot</string>
       </property>
      </widget>
     </widget>
     <widget class="QVideoWidget" name="video_2" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>451</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>120</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>영상 들어갈 부분</string>
       </property>
      </widget>
      <widget class="QVideoWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>921</width>
         <height>351</height>
        </rect>
       </property>
       <widget class="QTabWidget" name="tabWidget_3">
        <property name="geometry">
         <rect>
          <x>-40</x>
          <y>-30</y>
          <width>971</width>
          <height>621</height>
         </rect>
        </property>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_9">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QListView" name="listView_5">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>391</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Training</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>441</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Load</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>FileRoot</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>961</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Play</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>530</y>
        <width>411</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Save</string>
      </property>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>FileRoot</string>
       </property>
      </widget>
     </widget>
     <widget class="QVideoWidget" name="video" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>951</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>120</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>영상 들어갈 부분</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>450</y>
         <width>791</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../OneDrive/바탕 화면/갈매기/asdf.png</normaloff>../../OneDrive/바탕 화면/갈매기/asdf.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>430</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../OneDrive/바탕 화면/갈매기/pause.jpg</normaloff>../../OneDrive/바탕 화면/갈매기/pause.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QVideoWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>921</width>
         <height>391</height>
        </rect>
       </property>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>-40</x>
          <y>-50</y>
          <width>971</width>
          <height>621</height>
         </rect>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>160</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>동영상화면</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>530</y>
        <width>391</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Load</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>FileRoot</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Masking</string>
     </attribute>
     <widget class="QListView" name="listView_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>951</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Load</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QListView" name="listView_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>256</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>590</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Save</string>
      </property>
      <widget class="QListView" name="listView_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>256</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>550</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Masking</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
   <slots>
    <slot>PlayButtonM()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
